packages/dashboards/run.sh

```diff
@@ -74,8 +74,21 @@ if [ -z "${OPENCRVS_METABASE_MAP_REGION_NAME}" ]; then
   exit 1
 fi
 
+if [ -z "${OPENCRVS_METABASE_ADMIN_EMAIL}" ]; then
+  echo "Error: OPENCRVS_METABASE_ADMIN_EMAIL environment variable is not defined"
+  exit 1
+fi
+
+if [ -z "${OPENCRVS_METABASE_ADMIN_PASSWORD}" ]; then
+  echo "Error: OPENCRVS_METABASE_ADMIN_PASSWORD environment variable is not defined"
+  exit 1
+fi
+
 export MB_JETTY_PORT=${MB_JETTY_PORT:-4444}
 export MB_DB_FILE=/data/metabase/metabase.mv.db
+export OPENCRVS_METABASE_ADMIN_PASSWORD_SALT="635efd7e-ff74-4cc5-b460-95ee8523c5c5"
```

Comment:
if we hard code the salt then would it even work any security benefits?

Reply:
I agree that uuid might add extra layer of security, but as long as password comes from env variable, correct hash can be considered safe. Thoughts @rikukissa ?

Reply:
Think you are both right.. What if we regenerate the salt automatically every time we set the password? Not sure what parts of Metabase use it but from login point of view it should still work? I wouldn't want it to be user configurable either as it would be quite unconventional

Reply:
thanks @Zangetsu101 done in 26f10a183a10b734a3d34f412a8e6392a84d3ec4